docs/data-sources/resourcemanager_project.md

```diff
@@ -33,5 +32,16 @@ data "stackit_resourcemanager_project" "example" {
 
 - `id` (String) Terraform's internal data source. ID. It is structured as "`container_id`".
 - `labels` (Map of String) Labels are key-value string pairs which can be attached to a resource container. A label key must match the regex [A-ZÄÜÖa-zäüöß0-9_-]{1,64}. A label value must match the regex ^$|[A-ZÄÜÖa-zäüöß0-9_-]{1,64}
+- `members` (Attributes List, Deprecated) The members assigned to the project. At least one subject needs to be a user, and not a client or service account. This value is only considered during creation. Changing it afterwards will have no effect.
```

Comment:
Please delete this file, currently we don't want to show the `members` field in the documentation, as it was a feature that we introduced but had to rollback. So for now we always need to remove it manually when it is generated automatically

Reply:
only the change

Reply:
on both datasource and resource folders

Reply:
removed members section